core/test/vault-test.ts

```diff
@@ -0,0 +1,381 @@
+        await setupRpcUrlAndGetNetworks(LOCAL_RPC_URL);
+    });
+    it('Fetches not liquidatable vault', async () => {
+        await resetNetwork(15502000);
+
+        const vault = await fetchVault(TEST_NETWORK, 22025);
+        const vaultTransactionLiquidatable = (await getVaultTransaction(
+            TEST_NETWORK,
+            vault
+        )) as VaultTransactionNotLiquidated;
+        const expectedObject: VaultTransactionNotLiquidated = {
+            state: 'not-liquidatable',
+            collateralType: 'ETH-A',
+            address: '0x95dc6d7ED8991facab34159D8231e04247dfaf65',
+            network: TEST_NETWORK,
+
+            lastSyncedAt: new Date(),
+            nextPriceChange: new Date('2022-09-09T10:00:00.000Z'),
+
+            liquidationRatio: new BigNumber('1.45'),
+            collateralizationRatio: new BigNumber('3.204790430641710905476031356'),
+            proximityToLiquidation: new BigNumber('0.547552318511616565008415095'),
+            liquidationPenaltyRatio: new BigNumber('1.13'),
+            minimalAuctionedDai: new BigNumber('15000'),
+            id: 22025,
+
+            incentiveCombinedDai: new BigNumber('30168.839670692031996967035'),
+            incentiveRelativeDai: new BigNumber('29868.839670692031996967035'),
+            incentiveConstantDai: new BigNumber('300'),
+            grossProfitDai: new BigNumber('30168.839670692031996967035'),
+            netProfitDai: new BigNumber('32497.18533829802106158776170392239838650993987481484'),
+            stabilityFeeRate: new BigNumber('1.077990181020998014318011624'),
+            minUnitPrice: new BigNumber('1180.834957655172413793103448275'),
+            maximumProtocolDebtDai: new BigNumber('100000000'),
+            currentProtocolDebtDai: new BigNumber(0),
+            currentCollateralDebtDai: new BigNumber(0),
+            maximumCollateralDebtDai: new BigNumber(65000000),
+            transactionFeeLiquidationEth: new BigNumber('0.000669987000446658'),
+            transactionFeeLiquidationDai: new BigNumber('1.130545198449229148108694994096671174'),
+            debtDai: new BigNumber('32198315.88349647029073587039891649505768393987481484'),
+            collateralAmount: new BigNumber('60266.446946772373610407'),
+            currentUnitPrice: new BigNumber('1712.2106886'),
+            nextUnitPrice: new BigNumber('1712.2106886'),
+            initialDebtDai: new BigNumber('29868839.670692031996967035'),
+        };
+        compareVaultTransactionsNotLiquidated(expectedObject, vaultTransactionLiquidatable);
+    });
+    it('Fetches liquidated vault', async () => {
+        await resetNetwork(14955398);
+
+        const vault = await fetchVault(TEST_NETWORK, 7370);
+        const vaultTransaction = (await getVaultTransaction(TEST_NETWORK, vault)) as VaultTransactionLiquidated;
+        expect(vaultTransaction.state).to.eq('liquidated');
+        expect(vaultTransaction.pastLiquidations[0].auctionId).to.eq('ETH-A:726');
```

Comment:
Can we try to check this auction as well and at least compare the debt and the collateral amount?